kong/admin_service.go

```diff
@@ -232,7 +232,7 @@ func (s *AdminService) ListWorkspaces(ctx context.Context,
 	var workspaces []*Workspace
 	_, err = s.client.Do(ctx, req, &workspaces)
 	if err != nil {
-		return nil, fmt.Errorf("error updating admin workspaces: %v", err)
+		return nil, fmt.Errorf("error updating admin workspaces: %w", err)
```

Comment:
No, we don't always want to wrap. Please disable the linter that made you change this code.